addons/l10n_ro_edi_stock/data/template_etransport.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template id="l10n_ro_template_etransport">
+        <eTransport
+            xmlns="mfp:anaf:dgti:eTransport:declaratie:v2"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="mfp:anaf:dgti:eTransport:declaratie:v2"
+            t-att-codDeclarant="interface.l10n_ro_edi_stock_company_id.vat.upper().replace('RO', '')"
+            t-att-refDeclarant="declarant_ref">
+
+            <notificare t-att-codTipOperatiune="interface.l10n_ro_edi_stock_operation_type">
+                <corectie t-if="send_type == 'amend'" t-att-uit="interface.l10n_ro_edi_stock_document_uit"/>
+
+                <t t-foreach="interface.l10n_ro_edi_stock_move_ids" t-as="move">
+                    <t t-set="product" t-value="move.product_id"/>
+
+                    <bunuriTransportate
+                            t-att-codScopOperatiune="interface.l10n_ro_edi_stock_operation_scope"
+                            t-att-codTarifar="product.intrastat_code_id.code or '00000000'"
+                            t-att-denumireMarfa="product.name"
+                            t-att-cantitate="move.product_qty"
+                            t-att-codUnitateMasura="move.product_uom._get_unece_code()"
+                            t-att-greutateNeta="move.weight"
+                            t-att-greutateBruta="move.weight + sum(line.result_package_id.shipping_weight for line in move.move_line_ids if line.result_package_id)"
+                            t-att-valoareLeiFaraTva="product.list_price"/>
+                </t>
+
+                <partenerComercial
+                        t-att-codTara="partner.country_code"
+                        t-att-denumire="partner.name"
+                        t-att-cod="partner.vat.upper().replace('RO', '') if partner.vat else 'PF' if interface.l10n_ro_edi_stock_operation_type == '30' else None"/>
```

Comment:
this too.

for reference, you can see the template for cfdi in l10n_mx_edi (`cfdi.xml`).
Notice how a lot of the values there are pre-computed in python in `cfdi_values`